tests/kafkatest/services/console_consumer.py

```diff
@@ -256,9 +256,10 @@ def _worker(self, idx, node):
 
         consumer_output = node.account.ssh_capture(cmd, allow_fail=False)
 
-        self.init_jmx_attributes()
-        self.logger.debug("collecting following jmx objects: %s", self.jmx_object_names)
-        self.start_jmx_tool(idx, node)
+        with self.lock:
+            self._init_jmx_attributes()
```

Comment:
Maybe these this line and the one below could be moved into `start_jmx_tool`? If we go that far, we could do the locking in there as well.